### PR TITLE
[DUOS-2707][risk=no] Show null values in registration

### PR DESCRIPTION
**Pull request by rushtong**
### Addresses
https://broadworkbench.atlassian.net/browse/DUOS-2707

### Summary
Ensure that null values are returned in APIs that return a registration schema

----
Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/java/org/broadinstitute/consent/http/util/gson/GsonUtil.java [hunk 1]
```diff
@@ -23,6 +23,10 @@ public static Gson buildGson() {
     return gsonBuilderWithAdapters().create();
   }
 
+  public static Gson buildGsonNullSerializer() {
+    return gsonBuilderWithAdapters().serializeNulls().create();
```
Comment (fboulnois) [inline src/main/java/org/broadinstitute/consent/http/util/gson/GsonUtil.java:27]: Interesting, this is good to know.

**Review by fboulnois — APPROVED** [review]
lgtm 👍

**Review by hams7504 — APPROVED** [review]
lgtm 👍

File: src/main/java/org/broadinstitute/consent/http/util/gson/GsonUtil.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/consent/http/util/gson/GsonUtil.java:27]: Our default setting is to exclude null values so this should be a relatively infrequent use case.

**Pull request merged**